mkosi.conf.d/05-qemu-mem.conf

```diff
@@ -8,4 +8,4 @@ Distribution=|arch
 Distribution=|centos
 
 [Host]
-QemuMem=3G
+@QemuMem=3G
```

Comment:
what's `@`?

Reply:
Sets the default value which is used if no value is specified explicitly.